src/chart/layout/mod.rs

```diff
@@ -0,0 +1,400 @@
+use crate::{coord::Shift, style::IntoTextStyle};
```

Comment:
Just saw a few warning there, we need to make sure our code doesn't have warning